### DISCUSSION TITLE
Retrograde calculation for CYP intrinsic clearance

### DISCUSSION THREAD

**Discussion by regulus92** [Open-Systems-Pharmacology/Forum#827] [Questions & Problems]
Hello, everyone. I am stuck in a problem. I failed to work out the same values of CYP intrinsic clearance provided in some references according to a retrograde method (1). See the following tramadol example
![20190924100341](https://user-images.githubusercontent.com/45459593/65475865-a75e5400-deb3-11e9-81b9-20fe92725c08.jpg)
As I see, CL<sub>int,H</sub>=CL<sub>H</sub>×Q/(Q-CL<sub>H</sub>)×1/fu, and CL<sub>int,CYP</sub> = relative contribution×CL<sub>int,H</sub>/liver weight/CYP liver concentration. So from PK-Sim setting, I calculated CL<sub>int,2D6</sub> 0.38 μl/min/pmol. Where am I wrong? Besides, a 25 year-old European individual gives liver blood flow of 0.43 L/min in PK-Sim, which I think is too low. I can't find the source of a 99.15 L/h hepatic blood flow. 
![20190924103023](https://user-images.githubusercontent.com/45459593/65476708-6582dd00-deb6-11e9-928f-eaa32db404e0.png)
The following example (2) makes me more confused.
![20190924103617](https://user-images.githubusercontent.com/45459593/65476947-3ae55400-deb7-11e9-8100-9ba38981994b.jpg)
Simcyp software seems to have a retrograde calculation tool but I haven't find its algorithm.
Sorry for too many questions. I really appreciate your help. Thank you very much.

1. T'Jollyn H, Vermeulen A, Van Bocxlaer J. PBPK and its Virtual Populations: the Impact of Physiology on Pediatric Pharmacokinetic Predictions of Tramadol. AAPS J. 2018;21(1):8.
2. Johnson TN, Zhou D, Bui KH. Development of physiologically based pharmacokinetic model to evaluate the relative systemic exposure to quetiapine after administration of IR and XR formulations to adults, children and adolescents. Biopharm Drug Dispos. 2014;35(6):341-52.

**Comment by jzq90** [comment]
Hi,

0.43 L/min is hepatic artery blood flow. Below this part in PK-Sim, there is a portal vein blood flow rate 1.2 L/min. The sum of these two values is 1.63 L/min =97.8 L/h.

As for CLint,2D6, what's the value of CLH?

**Comment by regulus92** [comment]
Thank you for your reply. CL<sub>H</sub>=CL<sub>total</sub>-CL<sub>renal</sub>=25.5 L/h. Anyway, the second quetiapine example provides values that are far from our calculations.
